schemas/dab.draft.schema.json

```diff
@@ -175,6 +175,23 @@
             "enabled": {
               "type": "boolean",
               "description": "Allow enabling/disabling GraphQL requests for all entities."
+            },
+            "nested-mutations": {
+              "type": "object",
+              "description": "Feature flags for nested mutation operations",
+              "properties": {
+                "inserts":{
```

Comment:
nit: we should call this as `create` since that aligns with our existing nomenclature for actions, permission etc

Reply:
Renamed `insert` to `create`